packages/web/src/stores/admin.ts

```diff
@@ -20,7 +20,7 @@ export const useAdminStore = create<AdminStore>((set) => ({
       const { createdAt, updatedAt, ...rest } = res.data;
       admin = { ...rest, createdAt: dayjs(createdAt), updatedAt: dayjs(updatedAt) };
     } catch (error) {
-      if (!axios.isAxiosError(error) || error.status !== 401) {
+      if (!axios.isAxiosError(error) || ![401, 403].includes(error.response?.status as number)) {
```

Comment:
can you explain me what you did here ? @SpencerKaiser

Reply:
Changed the check to see if the error is a 403 or a 401 instead of just a 401... we really only want this to pop up when it's a 404 or a 500

Reply:
Got it thanks